gym-game2048/gym_game2048/2048_helpers.py

```diff
@@ -0,0 +1,36 @@
+import numpy as np
+from gym_game2048.envs.game_2048 import Game2048
+
+
+def show_obstacle_collision():
```

Comment:
This is a deterministic function that we can modify to show the results of 2 obstacles colliding, since the probability that the random actions will produce this effect before the game ends are pretty small